README.md

```diff
@@ -1,2 +1,23 @@
 # Apple_Clone
 Microverse Solo Project
+
+BUILDING WITH BACKGROUNDS AND GRADIENTS
+
+Follow the instructions atop the Google Homepage project to set up a Github repository for this project 
+
+
+Author:
+This project was executed by Kyankya Raymond student of Microverse.
+
+Technologies Used.
+HTML and CSS
```

Comment:
The readme file styling could be improved further, find some inspiration [here](https://github.com/othneildrew/Best-README-Template).